package.json

```diff
@@ -4,13 +4,14 @@
   "license": "MIT",
   "description": "generates a cryptographically strong random uid",
   "repository": "vercel/uid-promise",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "types": "lib/cjs/index.d.ts",
```

Comment:
Do you intend for Node.js consumers to always use CJS or should it use ESM when available?

Reply:
My thought was to keep it as it was yesterday, meaning node consumers would still continue to use CJS for now